common-java-libs/org.wso2.apk.apimgt.impl/src/main/java/org/wso2/apk/apimgt/impl/AbstractAPIManager.java

```diff
@@ -1172,9 +1172,10 @@ protected void populateDevPortalAPIInformation(String uuid, String organization,
         }
         api.setUriTemplates(uriTemplates);
         //CORS . if null is returned, set default config from the configuration
-        if (api.getCorsConfiguration() == null) {
-            api.setCorsConfiguration(APIUtil.getDefaultCorsConfiguration());
-        }
+        //Todo Need to implement for APK
```

Comment:
shall we follow this practice to keep track of todo items in apk repo? 
```suggestion
        // TODO (CrowleyRajapakse) Need to implement for APK
```

Reply:
removed the cors configuration setting